app/public/stylesheets/style.scss

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   Base
+   ========================================================================== */
+
+/**
+ * Base values
+ */
+
+$disclaimer-color: #6FC9C0;
```

Comment:
This is really cool. Don't ever name color variables related to the way they look. It can be a huge benefit as things get more complex.